pkg/controller/nodeset/node/controller.go

```diff
@@ -258,6 +263,46 @@ func (c *Controller) syncHandler(key string) error {
 }
 
 const (
+	NodeClassContentAnnotationKey = "nodeclass-content"
```

Comment:
should be more like nodeset.github.com/nodeclass-content

Reply:
or nodeset.k8s.io/....

Reply:
Done in df5f90c60393f9e8d5ebabebd2ee368e4b89521a